tests/chocolatey-tests/choco-uninstall.Tests.ps1

```diff
@@ -0,0 +1,43 @@
+Import-Module helpers/common-helpers
+
+Describe "choco uninstall" -Tag Chocolatey, UninstallCommand {
+    BeforeAll {
+        Initialize-ChocolateyTestInstall
+
+        New-ChocolateyInstallSnapshot
+    }
+
+    AfterAll {
+        Remove-ChocolateyTestInstall
+    }
+
+    Context "Uninstalling a side-by-side Package" {

```

Comment:
Could we skip this if version is below 1.2.0-alpha? That will allow these tests to be bypassed with current stable builds of Chocolatey.